src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/Formatters/IPackageReferenceContextInfoFormatter.cs

```diff
@@ -86,6 +89,12 @@ private IPackageReferenceContextInfoFormatter()
         protected override void SerializeCore(ref MessagePackWriter writer, IPackageReferenceContextInfo value, MessagePackSerializerOptions options)
         {
             writer.WriteMapHeader(count: 6);
+
+            WriteSerialize(ref writer, value, options);
```

Comment:
Why is this change made?

Reply:
Because this is a static helper method to write common fields between `IPackageReferenceContextInfoFormatter` and `ITransitivePackageReferenceContextInfo`.